cpp/include/ray/api.h

```diff
@@ -45,7 +45,9 @@ class Ray {
 
   /// Get a single object from the object store.
   /// This method will be blocked until the object is ready.
-  ///
+  /// We may get some exceptions when call Get, such as WORKER_DIED, ACTOR_DIED,
+  /// OBJECT_UNRECONSTRUCTABLE and TASK_EXECUTION_EXCEPTION.
```

Comment:
We can add this exception description like:
/// \Throws RayException if task or worker failed, or object is unreconstructable.